templates/c_quiz_question.html

```diff
@@ -5,24 +5,30 @@
     <title>Platform</title>
     <meta name="viewport" content=
         "width=device-width,initial-scale=1">
-    <link rel="stylesheet" href="{{ url_for('static', filename='css/c_quiz.css') }}">
+    <link rel="stylesheet" href="c_quiz.css">
   </head>
 
   <body>
     <!--navigation bar and title-->
     <div class="background">
-        <div class="nav">
-          <a href="/quiz" class="active"><strong>Take Quiz</strong></a>
-          <a href="/toolkit">View Toolkit</a>
-          <a href="/profile">Your Profile</a>
-          <a href="/home">Home</a>
+      <div class="nav">
+        <div class="profile-dropdown-menu">
+          <a id="navmenu">Your Profile</a>
+          <ul class="profile-dropdown-content">
+            <a href=c_profile.html>Edit Profile</a>

```

Comment:
Cannot add hyperlink to HTML file directly. Use a corresponding route defined in program1.py